sql/core/benchmarks/BloomFilterBenchmark-jdk17-results.txt

```diff
@@ -3,118 +3,190 @@ ORC Write
 Read a row from 100M rows:                Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------------------------------
-Without bloom filter, blocksize: 8388608           1244           1251          10         80.4          12.4       1.0X
-With bloom filter, blocksize: 8388608               504            513           8        198.4           5.0       2.5X
+Without bloom filter, blocksize: 8388608            673            695          28        148.5           6.7       1.0X
+With bloom filter, blocksize: 8388608               306            310           3        327.2           3.1       2.2X
 
 
 ================================================================================================
 Parquet Read
 ================================================================================================
 
 OpenJDK 64-Bit Server VM 17.0.2+8-LTS on Linux 5.13.0-1021-azure
-Intel(R) Xeon(R) Platinum 8171M CPU @ 2.60GHz
+Intel(R) Xeon(R) Platinum 8272CL CPU @ 2.60GHz
 Read a row from 100M rows:                 Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 -------------------------------------------------------------------------------------------------------------------------
-Without bloom filter, blocksize: 12582912           1384           1408          34         72.3          13.8       1.0X
-With bloom filter, blocksize: 12582912               900            955          49        111.1           9.0       1.5X
+Without bloom filter, blocksize: 12582912            678            681           4        147.4           6.8       1.0X
+With bloom filter, blocksize: 12582912               497            501           5        201.0           5.0       1.4X
 
 
 ================================================================================================
 Parquet Read
 ================================================================================================
 
 OpenJDK 64-Bit Server VM 17.0.2+8-LTS on Linux 5.13.0-1021-azure
-Intel(R) Xeon(R) Platinum 8171M CPU @ 2.60GHz
+Intel(R) Xeon(R) Platinum 8272CL CPU @ 2.60GHz
 Read a row from 100M rows:                 Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 -------------------------------------------------------------------------------------------------------------------------
-Without bloom filter, blocksize: 16777216           1468           1482          19         68.1          14.7       1.0X
-With bloom filter, blocksize: 16777216              1302           1313          15         76.8          13.0       1.1X
+Without bloom filter, blocksize: 16777216            675            738         102        148.2           6.7       1.0X
+With bloom filter, blocksize: 16777216              1071           1075           5         93.4          10.7       0.6X
```

Comment:
This looks anomaly in the GitHub Action server.

Reply:
Shall we rerun this to see if the anomaly will go away?

Reply:
Sure!